tests/lean/run/coe_to_fn.lean

```diff
@@ -21,6 +21,6 @@ local postfix `⁻¹` := equiv.inv
 def equiv.trans (f : α ≃ β) (g : β ≃ γ) : α ≃ γ :=
 ⟨g ∘ f, f⁻¹ ∘ g⁻¹⟩
 
-example (f : α ≃ β) := function.bijective f
+example (f : α ≃ β) := function.bijective.{u+1 v+1} f
```

Comment:
Issue 4: Why can't lean infer these universes? Or can it, because see:
Issue 5: Why can't lean unify `Π (m : has_coe_to_fun.domain f), has_coe_to_fun.codomain m` with `?m_1 → ?m_2`, when the `domain` and `codomain` functions are readily known to be constants and reduce in this context to `α` and `β`, respectively?